plugins/org.jkiss.dbeaver.ext.hana.ui/src/org/jkiss/dbeaver/ext/hana/ui/views/HANAConnectionPage.java

```diff
@@ -182,7 +182,7 @@ public void loadSettings() {
         super.loadSettings();
         DBPConnectionConfiguration connectionInfo = site.getActiveDataSource().getConnectionConfiguration();
         edition = HANAEdition.fromName(connectionInfo.getProviderProperty(PROV_PROP_EDITION));
-        portValue = CommonUtils.toString(connectionInfo.getHostPort(), site.getDriver().getDefaultPort());
+        portValue = CommonUtils.toString(connectionInfo.getHostPort()/*, site.getDriver().getDefaultPort()*/);
```

Comment:
We don't need default port anymore?

Reply:
All editions except GENERIC have there own rule to calculate the port from other fields or have a fixed port. 
GENERIC is not really an own edition, but a fallback that allows pasting already know host+port. The preset port was wrong in most cases as an on-premise HANA has an instance number in the range from 0 to 99 the actual port depends on that number.